src/wallet/rpcwallet.cpp

```diff
@@ -2431,7 +2443,7 @@ UniValue fundrawtransaction(const UniValue& params, bool fHelp)
 
         if (options.exists("feeRate"))
         {
-            feeRate = CFeeRate(options["feeRate"].get_real());
```

Comment:
We slipped up with the review for #7967, shouldn't have allowed using get_real here. Good that this is being replaced.